plugins/httpapi/mso.py

```diff
@@ -16,6 +17,16 @@
     a connection to MSO, send API requests and process the
     response.
 version_added: "1.2.0"
+options:
```

Comment:
Can ansible_httpapi_login_domain be accessed from the inventory?

Reply:
yes